automation/pages/authentication.page.ts

```diff
@@ -69,12 +72,36 @@ async fillRegistrationForm (
 
 async loginToAccount(){
     await this.loginButton.click();
-    await this.fillLoginData(process.env.EMAIL!, process.env.PASSWORD!);
+    await this.fillLoginData(process.env.EMAIL, process.env.PASSWORD);
     await this.accountLoginButton.click();
-}
+};
 
+async goToLoginPageAndFillLoginData(email: string, password: string){
```

Comment:
Pagal programavimo praktikas, metodai turi atlikti vieną konkrečią užduotį, ne daugiau, o šiuo atveju, tu sukūrei metodą, kuris naviguoja į login page'ą IR prisijungia, tas pats ir su registracijos metodu. 
Šiuo atveju metodas net susidurtų su problemom, nes kas jeigu tu būsi login page'e ir norėsi prisijungt prie paskyros? Atsakymas - Reikės atskiro metodo, nes šis metodas būnant login page'e dar tuo pačiu ieškos loginButton elemento, kurio nėra, testas fail'ins.

**Rekomendacija** - Išimčiau navigacijos logiką iš login bei registracijos metodų bei perkelčiau į pačius testus kaip atskirą step (guestpage.loginButton.click());